### ISSUE TITLE
ERC1155 support

### ISSUE THREAD

**Issue by barneyfoxuk** [PinataCloud/pinata-submarine#103]
**Is your feature request related to a problem? Please describe.**
I'm not 100% sure if this is a feature request or a bug as I haven't found a conclusive answer from documentation, but I am trying to verify ownership of an ERC1155 token to provide access of a submarined file. When trying a get a :Token not found. Check your network." response which leads me to believe only ERC721 is currently supported? 

**Describe the solution you'd like**
Adding support for ERC1155 token ownership would be really useful for my project.  As 1155 is now largely supported by NFT marketplaces etc. it would seem reasonable to me for you guys to add support, but obviously I don't know what level of technical work that would require from your side.

**Describe alternatives you've considered**
I originally built the project using ERC721 contract but it is far less optimised for what I am trying to achieve and would be a big step backwards for me.

**Additional context**
I'd be more than happy to help support any dev work you might need to build the feature / support on testing etc (I have 15 years commercial development experience)

**Comment by polluterofminds** [comment]
Hey @barneyfoxuk ERC-1155 is definitely supported. Can you share the unlock link if you're comfortable with that being public here on Github?

**Comment by barneyfoxuk** [comment]
Hey @polluterofminds - that's good to hear it's supported! I must just be doing something incorrectly...

Not 100% sure what you mean by unlock link so I'll post across a few details that might be useful:

Submarined file URL: https://limitedrun.mypinata.cloud/ipfs/QmT39RCo69LdcivPv4JeLxuax3hP4EfzQ5Nqvvz2L5XPFe

Contract I am trying to validate ownership against: https://rinkeby.etherscan.io/address/0xf7e61a1d795a496ba010cc33e02a896477846898

The account I am using for testing is 0xe77106486e5119CD9991Ea140f99aDFcbd88fF13 - when I can call the above contract's `balanceOf` method for token 1 I get back a value of one (which matches what I would expect to see, and the details in etherscan).

As I've just typed that out it's occurred to me the issue might be that when calling `submarine.verifyEVMNFT()` no token id is sent, which I would assumed would be required to know which token on the contract to check for ownership of. I haven't seen that in the docs as an option you can send but would be interested to hear from you if you think that might be the issue? I'll take a look through the `pinata-submarine` source code now to see if it's a valid option.

If it useful, the failing API call I see in my server logs looks like this:

```
url: 'https://app.submarine.me/api/verify',
data: '{"contractAddress":"0xf7E61a1d795a496bA010CC33E02A896477846898","network":"4","signature":"0x8556334f8a5db58b23252b96f21541d373da1e7ddd00823f6f456116d75e5a590adcc8ce56b4906680e95168435a3fe13e8a431a96c03c1699ee8c646da940581c","blockchain":"Ethereum","messageId":"1147ccce-8352-4f09-bd60-1a033d277f27","address":"0xe77106486e5119CD9991Ea140f99aDFcbd88fF13"}'}
```

From what I can that all looks correct (correct network for Rinkeby, user address, contract address etc) but let me know if anything jumps out as incorrect.

Thanks for your help man - appreciate it a lot!

**Comment by barneyfoxuk** [comment]
FYI I found you can send tokenId, which I have just tried, but still getting the `Token not found. Check your network` error back.

Request being sent for that looks like:
`url: 'https://app.submarine.me/api/verify',
data: '{"contractAddress":"0xf7E61a1d795a496bA010CC33E02A896477846898","network":"4","signature":"0x27249f17710329fc9c9a6ef3f920fc025c0b423ed593c703b27a6ab7d6b5f5450205f901966376818bf3fbef1c73975c02873c6b650e91a9c87dce1ad80bd3731b","blockchain":"Ethereum","messageId":"40dff76d-c665-48bf-afe8-81688f49b78d","address":"0xe77106486e5119CD9991Ea140f99aDFcbd88fF13","tokenId":1}'`

**Comment by barneyfoxuk** [comment]
Hey @polluterofminds - is there any other details I could give that would be useful? I'm still having the same issue with the token I am providing not being found.

Any help would be appreciated. Thanks.

**Comment by polluterofminds** [comment]
@barneyfoxuk normally tokenIDs on ERC1155 contracts are very long numbers. It's actually a required field in unlocking content based on ERC1155 NFT ownership. Do you have an OpenSea link I can look at?

**Comment by barneyfoxuk** [comment]
@polluterofminds Yeah sure thing. For the contract above an example on Opensea would be https://testnets.opensea.io/assets/rinkeby/0xf7E61a1d795a496bA010CC33E02A896477846898/1 . Looking at that page you can see it is owned by `0xe77106486e5119CD9991Ea140f99aDFcbd88fF13` which is the account I was trying to verify against.
